routes/web.php

```diff
@@ -1,7 +1,8 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
-
+use App\Http\Controllers\customersController;
+use App\Http\Controllers\changeLanguageController;
```

Comment:
```suggestion
use App\Http\Controllers\CustomersController;
use App\Http\Controllers\ChangeLanguageController;
```
sao class lại viết chữ đầu thường thế này :( xem lại quy phạm đặt tên nhé.